calico-cloud/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -0,0 +1,147 @@
+---
+description: Scan images in container registeries.
+---
+
+# Scan images in container registries
+
+Scan images in container registries at any time, on any infrastructure, including Kubernetes.
+
+## Value
+
+Add a layer of defense for images that don’t go through a pipeline (for example, third-party images), but are published to registries. If CVEs are missed in your build pipeline, you can catch them before they are deployed.
+
+## Concepts
+
+You can deploy the registry scanner wherever you want. Locally on a laptop. Anywhere. And it doesn’t have to be for Kubernetes.
+
+To use this option, all you need to do is:
+- Specify the registry paths to the images you want to scan
+- Provide permissions for the scanner to access your registries
+- Get a token for access to the Image Assurance API
+
+Based on the paths you specify, the scanner recursively scans all images in the registry once, and sends results to the Image Assurance dashboard in Manager UI. Because the scanner is not running continuously, it may take some time for results to display in the dashboard, especially if you have many images or large images.
+
+## Before you begin
+
+**Required**
+
+- You must have valid registry credentials
+
+**Supported registry platforms**
+
+- Amazon ECR
+- Azure Container Registry
+- Google GCR 
+
+**Limitations**
+
+- The registry scanner is available as an image (not using Tigera operator installation)
+- You can scan images from only one of the supported registry platforms/account per instance
+
+## How to
+
+- [Download the registry scanner](#download-the-registry-scanner)
+- [Setting up registry scanner](#set-up-registry-credentials)
+- [Scan images and send results to Calico Cloud](#scan-images-and-send-results-calico-cloud)
+- [Troubleshoot](#troubleshoot)
+- [Get registry scanner updates](#get-registry-scanner-updates)
+
+### Download the registry scanner
+
+The registry scanner comes in a docker image. Simply run `docker pull quay.io/tigera/image-assurance-registry-scanner:v1.10.4` in order to retrieve it.
+
+### Setting up registry scanner
+
+#### Credentials
+
+The registry scanner supports scanning all registries as long as the docker credentials are provided.
+Providing the `DOCKER_USERNAME` and `DOCKER_PASSWORD` lets the registry scanner retrieve images that the credentials have access to. Retrieving images 
+using these credentials will only retrieve images with tags.
+
+If you have a valid /.docker/config.json, you can also mount these while running the registry scanner.
+```
+docker run -v ~/.docker/config.json:/.docker/config.json quay.io/tigera/image-assurance-registry-scanner:v1.10.4
```

Comment:
See my [previous comment here](https://github.com/bmckercher123/docs/pull/3/files#r1341789632). We want to avoid using hard-coded IA version strings.

Reply:
Our release process doesn't tag our latest images as :latest or anything. Is that okay when pulling from the quay.io registry? I had this concern too.